src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,110 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
-
+import PropTypes from 'prop-types';
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
+
+import { getMovie } from '../../api';
+
+export const FindMovie = ({ addMovie }) => {
+  const [value, setValue] = useState('');
+  const [movie, setMovie] = useState('');
+  const [searchError, setSearchError] = useState(false);
+
+  const onChangeValue = (event) => {
+    setValue(event.target.value);
+    setSearchError(false);
+    setMovie('');
```

Comment:
'movie' is an object. Initially, it can be either null or {}, but definitely not a blank string

Reply:
Fixed